src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,150 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private int size;
+
```

Comment:
make sure that you have checked the common mistakes : https://github.com/mate-academy/jv-linked-list/blob/master/checklist.md